src/wallet/db.cpp

```diff
@@ -3,12 +3,12 @@
 // Distributed under the MIT software license, see the accompanying
 // file COPYING or http://www.opensource.org/licenses/mit-license.php.
 
-#include <wallet/db.h>
```

Comment:
The module header is always included first, I think

Reply:
Fixed!